recipes/natural/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "natural" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 18c83662d2d33fd7e6eee4e3b0d7366e1ce86225664e3127a2aaf0a3233f7df2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - six
+  run:
+    - python >=3.6
+    - six
+
+test:
+  imports:
+    - natural
+
+about:
+  home: "https://github.com/tehmaze/natural"
+  license: MIT
+  license_family: MIT
+  # license file manually packaged. See https://github.com/tehmaze/natural/issues/22
+  license_file: LICENSE
```

Comment:
I added a comment since I opened an issue upstream to include the license file in the source distribution (https://github.com/tehmaze/natural/issues/22). The comment is to act as a visual reminder to check in future releases of the package if the license file is included. Once it is then you can remove the manually packaged copy and the comment.